pkg/ds/farm.go

```diff
@@ -565,13 +565,17 @@ func (dsf *Farm) spawnDaemonSet(
 	ctx, cancel := context.WithCancel(ctx)
 
 	desiresCh := ds.WatchDesires(ctx, updatedCh, deletedCh)
-
+	config := dsFields.Manifest.GetConfig()
+	// defaults to false if not set
+	useHealthService, _ := config["use_health_service"].(bool)
```

Comment:
this will crash the daemon set farm if it's a string for example right?

Reply:
Right, I could put it in a switch that checks for type and only set if the type is bool, otherwise default to false like this https://play.golang.org/p/KCvc9DgKfHI

This doesn't panic if the key in the config map doesn't exist either, the value will be nil when trying to access a key that doesn't exist.